lib/backend-operations/ops-cloud-api.js

```diff
@@ -148,7 +148,11 @@ function downloadLambdaFunction(projectInfo, awsDetails, lambdaFunction, callbac
     const lambdaClient = awsClient.Lambda(awsDetails, lambdaFunction.attributes.region)
     lambdaClient.getFunction(getParams, (err, data) => {
         if(err){
+            console.log(chalk.red('error get lambda function details for ' + lambdaFunction.name))
```

Comment:
I would say
'error on getting lambda function details for:' + lambdaFunction.name + ' function'